app/bundles/CoreBundle/Views/Helper/list_actions.html.twig

```diff
@@ -8,7 +8,7 @@
 {# Workaround to get variables from action_button_helper here #}
 {{ restore('action_button_helper') }}
 
-{% if item is iterable %}
+{% if item is iterable and item[0] is not defined %}
```

Comment:
I'm not a fan of this. The item array does not have to start with 0. Sometimes the IDs from the database are used as array keys. Can this check for the actual property it needs?
```twig
{% if item is iterable and item.id is defined and item.name is defined %}
```

Reply:
I want to check if `item` is an object and not an array. This particular line mimics what PHP does, only Twig doesn't have a complete check like php does. I think this is much cleaner than checking for particular keys.

Reply:
There's no reason I can see that the item would contain array keys. Can you give me an example?

Reply:
Ah, hold on, you're right. I was confusing this check with one I got rid of in another template. My bad, sorry!
Wil make changes and set up a new PR. It will have to wait until Thursday though, as I won't be working tomorrow (Wednesday)
Thanks for catching this!

Reply:
Made a new PR for the changes in this review

Reply:
[Made a new PR for the changes in this review](https://github.com/mautic/mautic/pull/12017)